doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -92,6 +92,7 @@ Matter
     8 dBm for nRF52840, 3 dBm for nRF5340, 20 dBm for all devices with FEM enabled, and 0 dBm for sleepy devices.
 
 * Fixed the issue of connection timing out when attaching to a Wi-Fi access point that requires Wi-Fi Protected Access 3 (WPA3).
+* Removed deprecated Android Chip Tool sections in the documentation as Android Chip Tool is no longer maintained.
```

Comment:
Hmm, it's not officially not maintained.. i would rather mentioned sth like:

* Removed support for Android CHIP Tool from the documentation and release artifacts. It is recommended to use the development tool CHIP Tool or mobile applications from publicly available Matter Ecosystems.